README.md

```diff
@@ -127,6 +122,7 @@ If you need to change parameters and re-run a workflow, first do:
     cylc clean <workflow-name>
     ```
    - [ ] Then, proceed to install, play, and open the TUI
+   - [ ] To rerun in one line: ```cylc stop --now <workflow-name> && cylc clean <workflow-name> && cylc install -n <workflow-name> ./config/cylc_hpc && cylc play <workflow-name> && cylc tui <workflow-name>```
```

Comment:
you might want to include `cylc validate` in here – that checks your flow file before you start